### DISCUSSION TITLE
投稿記事の情報一覧ダウンロード

### DISCUSSION THREAD

**Discussion by hanaseleb** [increments/qiita-discussions#601] [要望・アイデア・不具合]
過去に自分が投稿した記事の情報を一覧でダウンロードできる機能が欲しい。APIでやれるでしょっていうのはありますが、機能として実装されていると便利だなと思います。投稿記事は転職活動などで実績として出したい人もいるでしょうし。

記事タイトル、閲覧数、投稿日、更新日、いいね数、ストック数、タグなどがCSVやExcelでダウンロードできるといいなと思います。

**Comment by tanaka-asami415** [comment]
@hanaseleb 
リクエストありがとうございます🙏

大変恐れ入りますが、現在のところ、ご記載いただいた機能を実装する可能性は低そうです🙇
APIやCLIをご活用いただけると幸いです。
